packages/table/src/Table.js

```diff
@@ -219,13 +220,15 @@ class Table extends React.Component {
       ? "wg-small-arrow-top"
       : "wg-small-arrow-bottom";
     return (
-      <Button
-        icon={icon}
-        onClick={e => {
-          this.expandToggleClicked(rowKey);
-          e.stopPropagation();
-        }}
-      />
+      <ButtonContainer>
```

Comment:
`ButtonContainer` is a `div` with flex and justify-content only.
I think it'd be better to set these styles on `BodyCell` component.

This way we would have this:

```
<div className="class-name-x">
  <Button
    icon={icon}
    onClick={e => {
      this.expandToggleClicked(rowKey);
      e.stopPropagation();
    }}
  />
</div>
```

And `BodyCell` would be like this:
```
export const BodyCell = styled.td`
  text-align: ${textAlign};
  background-color: ${cellBg};

  .class-name-x {
    display: flex;
    justify-content: flex-end;
  }
`;
```

Reply:
From this comment I noticed that we have this `text-align` rule already, which can be set by the `align` prop. So you can simple use this at line 188: 
```
<BodyCell className="cell" align="right" {...rowProps}>
```